slsim/Sources/supernovae.py

```diff
@@ -39,33 +59,34 @@ def __init__(
         :param mag_zpsys: Optional, AB or Vega (AB default)
         :type mag_zpsys: str
         :param cosmo: Cosmology for absolute magnitude
-        :type cosmo: `~astropy.cosmology`
+        :type cosmo: `~astropy.cosmology
```

Comment:
just checking, the ' at the end should be removed?

Reply:
I must have deleted that by accident, will fix